.changeset/twenty-pears-admire.md

```diff
@@ -0,0 +1,15 @@
+---
+'@commercetools-frontend/sdk': patch
+---
+
+Add support for requesting specific `claims` to be included in the exchange JWT sent from Merchant Center API to an external API.
+Currently we only support requesting a custom claim with logged in user's permissions.
+
+```js
+actions.forwardTo.get({
+  exchangeTokenClaims: ['permissions'],
```

Comment:
```suggestion
  includeUserPermissions: true,
```

Reply:
Updated in this commit: 0f7be5b44a05e5012364c8ead78856d761445fd4